### DISCUSSION TITLE
Where is the repository for espoj files (Microsoft.VisualStudio.JavaScript.Sdk and its templates)?

### DISCUSSION THREAD

**Discussion by cremor** [dotnet/aspnetcore#56111] [Q&A]
I'm wondering where I can find the repository for espoj files (Microsoft.VisualStudio.JavaScript.Sdk and its templates).
The [NuGet package](https://www.nuget.org/packages/Microsoft.VisualStudio.JavaScript.SDK) only links to a survey.

@Forgind said [here](https://github.com/dotnet/sdk/issues/41017#issuecomment-2135848889) that it's https://github.com/microsoft/TypeScript but I can't find anything related there.

Cc @joj because of [a comment](https://github.com/dotnet/sdk/issues/39152#issuecomment-1974228678) from @baronfel

**Comment by Forgind** [comment]
If you're curious why I thought that might be the right repo, I went to nuget.org and searched for that SDK:
https://www.nuget.org/packages/Microsoft.VisualStudio.JavaScript.SDK

The two owners for that are Microsoft and TypeScriptTeam.

But I also wasn't 100% sure, so rather than move it, I pinged someone who had contributed to the repo recently, though they didn't respond unfortunately.

**Reply by baronfel** [reply]
It's not OSS and it's not owned by the TypeScript team, though @joj's team is the owner.

**Reply by Forgind** [reply]
Is there another repo I should move esproj issues to, or should I assign joj?

**Reply by baronfel** [reply]
They mostly have surfaced in dotnet/sdk, but I think the correct answer for now is to use VS Developer Community until/unless the esproj team provides a more GitHub-friendly point of contact.

**Reply by cremor** [reply]
> I think the correct answer for now is to use VS Developer Community until/unless the esproj team provides a more GitHub-friendly point of contact.

That's fine for bug reports, but what about getting information (like release notes) or asking general questions?
